### PR TITLE
Add blog post for restic 0.17.1

### PR DESCRIPTION
**Pull request by MichaelEischer**

### CODE REVIEW

File: content/blog/restic-0.17.1-released.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Restic 0.17.1 Released"
+date: 2024-09-03T00:00:00+02:00
+---
+
+We are happy to announce the release of [restic 0.17.1](https://github.com/restic/restic/releases/v0.17.1)!
+
+This release contains various bugfixes and small enhancements. The most important are the following:
```
Comment (rawtaz) [inline content/blog/restic-0.17.1-released.md]:
```suggestion
This release contains various bug fixes and small enhancements. The most important ones are the following:
```

File: content/blog/restic-0.17.1-released.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- The backup command now always stores files and directories even if some metadata could not be retrieved.
```
Comment (rawtaz) [inline content/blog/restic-0.17.1-released.md]:
```suggestion
- The backup command now also stores files and directories for which some metadata could not be retrieved.
```

**Review by rawtaz — CHANGES_REQUESTED** [review]

File: content/blog/restic-0.17.1-released.md (hunk 1 above)
Comment (MichaelEischer) [reply content/blog/restic-0.17.1-released.md]: changed. Thanks!

File: content/blog/restic-0.17.1-released.md (hunk 2 above)
Comment (MichaelEischer) [reply content/blog/restic-0.17.1-released.md]: also changed.

**Review by rawtaz — APPROVED** [review]

**Pull request merged**